src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/ConfigCommands/ConfigCommand.cs

```diff
@@ -0,0 +1,278 @@
+                int exitCode;
+
+                var args = new ConfigPathsArgs()
+                {
+                    WorkingDirectory = parseResult.GetValue(workingDirectory_Option),
+                };
+
+                try
+                {
+                    ConfigPathsRunner.Run(args, getLogger);
+                    exitCode = 0;
+                }
+                catch (Exception e)
+                {
+                    LogException(e, getLogger());
+                    ShowHelp(parseResult, cmd);
+
+                    exitCode = 1;
+                }
+                return Task.FromResult(exitCode);
+            });
+        }
+
+        private static void RegisterOptionsForCommandConfigGet(CliCommand cmd, Func<ILogger> getLogger)
+        {
+            var allOrConfigKey_Argument = new CliArgument<string>(name: "all-or-config-key")
+            {
+                Arity = ArgumentArity.ZeroOrOne,
+                HelpName = Strings.ConfigGetAllOrConfigKeyDescription,
+                Description = Strings.ConfigGetAllOrConfigKeyDescription
+            };
+            cmd.Add(allOrConfigKey_Argument);
+            var workingDirectory_Argument = new CliOption<string>(name: "--working-directory")
+            {
+                Arity = ArgumentArity.ZeroOrOne,
+                Description = Strings.ConfigPathsWorkingDirectoryDescription,
+            };
+            cmd.Add(workingDirectory_Argument);
+            var showPath_Option = new CliOption<bool>(name: "--show-path")
+            {
+                Arity = ArgumentArity.Zero,
+                Description = Strings.ConfigGetShowPathDescription,
+            };
+            cmd.Add(showPath_Option);
+            var help = new HelpOption()
+            {
+                Arity = ArgumentArity.Zero
+            };
+            cmd.Add(help);
+
+            // Create handler delegate handler for cmd
+            cmd.SetAction((parseResult, cancellationToken) =>
+            {
+                int exitCode;
+                var args = new ConfigGetArgs()
+                {
+                    AllOrConfigKey = parseResult.GetValue(allOrConfigKey_Argument),
+                    WorkingDirectory = parseResult.GetValue(workingDirectory_Argument),
+                    ShowPath = parseResult.GetValue(showPath_Option),
+                };
+
+                try
+                {
+                    ConfigGetRunner.Run(args, getLogger);
+                    exitCode = 0;
+                }
+                catch (Exception e)
+                {
+                    LogException(e, getLogger());
+                    ShowHelp(parseResult, cmd);
+
+                    exitCode = 1;
+                }
+                return Task.FromResult(exitCode);
+            });
+        }
+
+        private static void RegisterOptionsForCommandConfigSet(CliCommand cmd, Func<ILogger> getLogger)
+        {
+            var configKey_Argument = new CliArgument<string>(name: "config-key")
```

Comment:
There's a lot of repeated args in this method and the below one and it makes me think w e can easily have a typo. 
Is it worth merging some of these methods to ensure we don't end up with a typo?

Reply:
I'll try and see if I can move these CliOption and CliArgument outside of the functions, so we have only one definition, and just add them when needed. There are more complex commands that will have a lot of options, this would make it easier to read too.